GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx

```diff
@@ -61,8 +84,42 @@ const RegistrationForm = () => {
         if (e.target.value !== "annet") setDessertInput("");
     } 
 
+    const onSubmit1 = (e : React.FormEvent<HTMLFormElement>) => {
```

Comment:
onSubmit1? are there multiple onSubmit-s here?
If there are they should probably have a name describing what the submit 😉

Reply:
fair, that should be changed into something more understandable